rfcs/0008-node-discovery/0008-node-discovery.md

```diff
@@ -0,0 +1,92 @@
+---
+Number: 8
+Category: Standards Track
+Status: Draft
+Author: Linfeng Qian / JinYang Jiang
+Organization: Nervos Foundation
+Created: 2018-11-28
+---
+
+# CKB Node Discovery Protocol
+
+CKB node discovery protocol mainly refers to [Satoshi Client Node Discovery][0]. The differences between them are summarized below:
+
+* The node version number is included in the `GetNodes` message.
+* The `Nodes` message is used to periodically broadcast all nodes currently connected.
+* We use `multiaddr` as the format of node addresses  (There is no `/p2p/` field here and if it's considered as *misbehavior*, a low score will be given.)
+
+Every time client startup, if PeerStore's address list is empty, it will try to issue DNS requests to initialize address list, if DNS requests failed it will fallback to hard-coded address list.
+
+## Discovery Methods
+### DNS Addresses
+When first time startup (bootstrap stage), if discovery service is needed, local node then issues DNS requests to learn about the addresses of other peer nodes. The client includes a list of host names for DNS services that are seeded. DNS server addresses can be replaced by command line arguments.
```

Comment:
```suggestion
At the first time startup (bootstrap stage), if the discovery service is needed, the local node will issue DNS requests to learn about the addresses of other peer nodes. The client includes a list of seed hostnames for DNS services.
```

Reply:
Removed `DNS server addresses can be replaced by command line arguments.` which is an implementation detail, not protocol.